src/app/pages/storage/modules/pool-manager/components/manual-disk-selection/manual-disk-selection.component.ts

```diff
@@ -58,10 +59,13 @@ export class ManualDiskSelectionComponent implements OnInit {
         }
       }
 
+      this.hideAddVdevButton = this.data.vdevsLimit && vdevs.length >= this.data.vdevsLimit;
```

Comment:
Setting `hideAddVdevButton` doesn't belong to something called `isSaveDisabled$`.
You need to create a new observable for it.